apply_license.py

```diff
@@ -76,76 +75,79 @@ def match_patterns(s, patterns):
             return True
     return False
 
+def skip_path(fullpath):
+    return match_patterns(fullpath, excluded_paths)
 
-def force_include(fullpath):
-    if match_patterns(fullpath, include_patterns):
-        return True
-    return False
-
-def skip_filename(fullpath, basename):
-    if basename in exluded_files:
-        return True
-
-    if match_patterns(fullpath, exluded_patterns):
-        return True
+def skip_filename(fullpath):
+    return match_patterns(fullpath, excluded_files)
```

Comment:
Why not pass in the filename as opposed to the full path? (Easier to avoid mistakes)

Reply:
Done!